src/components/inputs/Selector.tsx

```diff
@@ -125,9 +125,6 @@ export default function Selector({
               cursor: disabled ? 'auto' : 'pointer',
               borderRadius: '4px',
               padding: '4px 8px',
-              ':hover': {
-                backgroundColor: (theme) => theme.color.gray[100],
-              },
```

Comment:
Removing this has removed the background on hover from the assignee dropdown in the sidebar. Can we fix this to what it is like in the production branch now? 
See the SS below - 
![Screenshot from 2024-07-11 12-13-10](https://github.com/copilot-platforms/tasks-app/assets/38468429/f2844084-0a1a-41a2-a5c6-b885c798bfe0)

Reply:
While fixing this, can you also add a patch for below issue? 
The assignee avatar icon's letter "S" is not properly placed. May be it needs some padding changes. If it is avatar image then it looks fine though.